src/main/java/com/browserstack/automate/ci/jenkins/BrowserStackCredentials.java

```diff
@@ -45,6 +45,7 @@
     private static final String CREDENTIAL_DISPLAY_NAME = "BrowserStack";
     private static final String OK_VALID_AUTH = "Success";
     private static final String ERR_INVALID_AUTH = "Invalid username or access key!";
+    private static final String ERR_BS_BLOCKED = "Unable to connect to BrowserStack!";
```

Comment:
ERR_BS_BLOCKED  -> ERR_HUB_UNREACHABLE in case of etimedout or network errors. 


in all other cases  should it return the message from the http response?  ( be it invalid creds / plan expiry / rate limit exceeded / internal errors)

Reply:
Updated variable name